Add initial configuration for webpack-dev-server

Jefferson227 opened the pull request:
# Description

This pull request adds an initial configuration in the `package.json` that avoids the `webpack-dev-server` to display the directory list in the `localhost:8080` (what happened to me) and makes it display the actual content from the index page when executed the `npm run start` command is executed.

## Type of change

This is a:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (fixes or update of an existing feature)

## Checklist:

By making this contribution I ensure that:

- [x] I have read the [contribution guide](https://github.com/DevTony101/js-todo-app-indexed_db/blob/master/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [README file](https://github.com/DevTony101/js-todo-app-indexed_db#features) (if this PR introduces a new feature)
- [x] My changes generate no new warnings

DevTony101 commented:
Hi there! Thank you for your contribution and showing interest towards this project! This indeed is an interesting feautre but it does not appears to work on my local development server as is still displaying the directory list, maybe it would work if the options were in the `webpack.config.js`?

Jefferson227 commented:
Hi @DevTony101 , yes, you're right. When I tested again with the configuration on `package.json` it didn't work and displayed the directories again. I just moved the configuration to `webpack.config.js` and changed the `contentBase` to `./dist` and it worked again.

DevTony101 reviewed (approved):
Excellent! Everything works as expected. Thanks again for your contribution 👍🏽
